packages/@aws-cdk/aws-servicecatalog/test/integ.product.ts

```diff
@@ -20,6 +29,9 @@ new servicecatalog.CloudFormationProduct(stack, 'TestProduct', {
     {
       cloudFormationTemplate: servicecatalog.CloudFormationTemplate.fromAsset(path.join(__dirname, 'product2.template.json')),
     },
+    {
+      cloudFormationTemplate: servicecatalog.CloudFormationTemplate.fromStack(new TestProductStack(stack, 'SNSTopicProduct')),
+    },
```

Comment:
Can you actually add a second ProductStack Version here? I want to make sure multiple Assets are correctly rendered in this test.